api/src/main/resources/plugins/sparkapp_tpl.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: 
+  namespace: pnda
+spec:
+  type: Scala
+  mode: cluster
+  image: "gcr.io/spark-operator/spark:v2.4.4"
+  imagePullPolicy: Always
+  mainClass: 
+  mainApplicationFile: 
+  sparkVersion: "2.4.4"
+  restartPolicy:
+    type: Never
+  volumes:
+    - name: "sparkvol"
+      persistentVolumeClaim:
+        claimName: pnda-deployment-manager
+  driver:
+    cores: 1 
+    coreLimit: "1200m"
+    memory: "512m"
+    labels:
+      version: 2.4.4
+    serviceAccount: pnda
+    volumeMounts:
+      - name: "sparkvol"
+        mountPath: /data/stage
+  executor:
+    cores: 1
+    instances: 2
+    memory: "512m"
```

Comment:
We should be able to customize some of this values in the PNDA app properties.json file.

Reply:
We can add this feature in a future PR.